### PR TITLE
Feature/oops 4615/create distribution package

### PR DESCRIPTION
**Pull request by grzark**
Chocolatey package scripts for the MSI and VSIX. This is dependent on the "Plex-Updates" PR so this is staying at a draft until that is merged.

### CODE REVIEW

File: chocolatey/_TODO.txt [hunk 1]
```diff
@@ -0,0 +1,131 @@
+﻿TODO
```
Comment (numo16) [inline chocolatey/_TODO.txt]: This file can be deleted from the repo if we've completed everything on it/

File: chocolatey/tools/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿
+Note: Include this file if including binaries you have the right to distribute. 
+Otherwise delete. this file.
```
Comment (numo16) [inline chocolatey/tools/LICENSE.txt]: This should be updated or removed, per the above note

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+trigger:
```
Comment (numo16) [inline azure-pipelines.yml]: Hadn't considered the pipeline yaml in our github repos before, not sure if something to be concerned about. Thoughts @ajhall @debben ?

**Review by numo16 — COMMENTED** [review]
First pass review complete

File: azure-pipelines.yml (hunk 3 above)
Comment (ajhall) [reply azure-pipelines.yml]: As long as we control all the branches in this repo it *should* be OK... but I think we should segregate our GitHub repos from our internal build agents just for safety. A malicious pipeline that somehow makes it onto any branch in our repos would run on a build agent our internal network, and we shouldn't leave that possibility open.

Comment (grzark) [reply azure-pipelines.yml]: I'll remove it for now

**Review by numo16 — APPROVED** [review]

**Pull request merged**